app/workers/education_form/create_daily_fiscal_year_to_date_report.rb

```diff
@@ -134,7 +134,10 @@ def create_csv_data_block(region, submissions, submissions_total)
       application_types = EducationBenefitsClaim::APPLICATION_TYPES
 
       application_types.each_with_index do |application_type, i|
-        on_last_index = i == (application_types.size - 1)
+        next if application_type == 'vettec' && region != :eastern
+
+        on_last_index = i == (application_types.size - (region == :eastern ? 1 : 2))
```

Comment:
why is this necessary?

Reply:
:heavy_plus_sign:  :point_up:

Reply:
Done to support the following requirement:
1) A new benefit type `vettec` needs to be added **ONLY** to be made available for the eastern region
2) `vettec` has to be the last row under the eastern region, which also holds counts for the 22-0993 and 22-1990n as they don't have individual benefits.

Reply:
Sample CSV daily report screenshot 
<img width="1674" alt="screen shot 2019-01-30 at 10 35 47 am" src="https://user-images.githubusercontent.com/11605414/51992394-eea65900-247a-11e9-80da-66610c29a6eb.png">
attached.

Reply:
@lihanli : As per our discussion, code changes are done to remove the workaround for index calculation. `vettec` benefit type is the last row for all the regions. Sample latest screenshot updated as shown for the forms submitted & processed.
<img width="1672" alt="screen shot 2019-01-30 at 5 56 41 pm" src="https://user-images.githubusercontent.com/11605414/52018690-3a77f300-24b9-11e9-9522-18fc8581ee40.png">

use case created to generate the report for the following forms:
1) Form 22-1990 (chapter1606) - eastern
2) Form 22-1990n - eastern
3) Form 22-0993 - western
4) Form 22-0994 (vettec) - eastern